timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -120,7 +125,6 @@ private void registerLeaderElectionService(TimeLockServerConfiguration configura
                 localPaxosServices.ourAcceptor(),
                 localPaxosServices.ourLearner()));
         environment.jersey().register(new NotCurrentLeaderExceptionMapper());
```

Comment:
Ah, I left this here because it was related to the Paxos election stuff, while the others were not. I'm fine either way.